app/views/layouts/application.html.erb

```diff
@@ -67,18 +66,33 @@
                 <li class="nav-dropdown-item"><a href="/forms/new">Forms</a></li>
               </ul>
             </li>                
-            <li>
-              <a href="#contact" class="cdc-navbar-item"><i class="fa fa-bell item-navbar-icon" aria-hidden="true"></i>
+            <li class="dropdown">
+              <a href="#notifications" class="dropdown-toggle cdc-navbar-item" id="notifications-dropdown" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><i class="fa fa-bell item-navbar-icon" aria-hidden="true"></i>
               Alerts
-              <span class="alerts-badge">1</span></a>
-            </ul>
+              <% if current_user.notifications.exists? and current_user.notifications.where('read' => false).count > 0 %>
```

Comment:
I would expect all of this to be in the react widget not in the rails template.  The react template should take care of displaying the number of alerts